clubhouse/clubhouse.py

```diff
@@ -87,10 +87,11 @@ def wrap(self, *args, **kwargs):
             return func(self, *args, **kwargs)
         return wrap
 
-    def __init__(self, user_id='', user_token='', user_device=''):
+    def __init__(self, user_id='', user_token='', user_device='', headers=None):
         """ (Clubhouse, str, str, str) -> NoneType
         Set authenticated information
         """
+        self.HEADERS = dict(headers) if isinstance(headers, dict) else dict(self.HEADERS)
```

Comment:
the rest of original headers will be removed when you add the `headers` parameter.